kudu/kudu.py

```diff
@@ -0,0 +1,73 @@
+import urllib2
```

Comment:
Also we should make the script both python 2.x and 3.x compliant. It'll currently not work with python 3. 

I'd recommend using the `requests` module (`pip install requests`): http://docs.python-requests.org/en/master/ which handles it for you. But otherwise you'd have to do something like this with the imports:

```
if sys.version_info[0] > 2:
    import urllib.request as urllib
    import urllib.error as urlerror
    from urllib.parse import urlparse
else:
    import urllib2 as urllib
    import urllib2 as urlerror
    from urlparse import urlparse
```